Created event for suborbital champions cohort (#240)
Openscapes/website-new: events/2026-06-03-suborbital-champions/index.qmd

---
title: "NASA Openscapes Champions Program, Spring 2026"

> description: "An Openscapes Cohort to learn open science approaches for your daily workflows with NASA airborne and other suborbital data"

Reviewer: ```suggestion
description: "An Openscapes Cohort to learn open science approaches for your daily workflows  with NASA airborne and other suborbital data"
```

Data Management Plans, GitHub, “Forking as a Worldview” mindset). Working
together in the cohort will help to lower barriers for a better workflow
documentation by developing shared skills and strengthened relationships.

## Program details

[Openscapes Champions](https://www.openscapes.org/champions/) is a
remote-by-design mentorship program for researchers to explore open data science
practices. [Core lessons
include](https://openscapes.github.io/series/what-to-expect.html#cohort-calls)
open mindset, GitHub for publishing and
project management, coding and data strategies, and team culture.
To support EVS-4 teams, this cohort will also focus on documentation strategies for suborbital data.

The 2026 NASA Suborbital Openscapes Champions Cohort will run in June 2026. We
will meet as a Cohort via Zoom four times over one month for 1.5 hours, on
alternating Wednesdays or Thursdays. We propose the following three options:

 - **When**:
   - Wednesdays at 10-11:30 am PT  Dates: June 3, 10, 17, 24

<!--- on alternating Wednesdays in April - June, 2026.

-   **Dates**: April 15, 29, May 13, 27, June 10.

-   **Times**: 14:00 - 15:30 GMT.

-   **Where**: remotely, via Zoom. --->

> -   **Who**: EVS-4 and SoAR teams.

Reviewer: ```suggestion
-   **Who**: Researchers (postdoctoral fellows, graduate students, principal investigators, other academic researchers) and people supporting research with NASA airborne and suborbital data. Preference will be given to EVS-4 and SoAR teams if we reach capacity. 
```

---

> title: "NASA Suborbital Champions Program 2026"

Reviewer: ```suggestion
title: "NASA Openscapes Champions Program, Spring 2026"
```

> description: "An Openscapes Cohort to learn open science approaches for your daily workflows"

Reviewer: ```suggestion
description: "An Openscapes Cohort to learn open science approaches for your daily workflows with NASA airborne and other suborbital data"
```

author:
  - name: Openscapes and NASA
date: 2026-06-03
categories:
  - event 
  - champions
image: nasa-openscapes.png
bibliography: references.bib
---

## Join us to do better science in less time, together

> **This is a professional development and leadership opportunity for NASA

Reviewer: ```suggestion
**This is a professional development and leadership opportunity for
```

> suborbital affiliated researchers and people supporting NASA suborbital

Reviewer: ```suggestion
people doing and supporting research using NASA Earthdata collected from suborbital instruments**
```

> people doing and supporting research using NASA Earthdata collected from suborbital instruments**

Reviewer: ```suggestion
people doing and supporting research using NASA Earthdata collected from suborbital sampling**
```

> research** to explore open data science practices and make incremental and

Reviewer: ```suggestion
to explore open data science practices and make incremental and
```

sustainable change, no matter where you are starting from. The 2026 NASA
Suborbital Openscapes Champions Cohort will run in June 2026. We will meet as
a Cohort via Zoom four times over one month.
<!---The 2026 Champions Cohort will
run from **April 15th to June 10th, 2026**, meeting biweekly 14:00 - 15:30 GMT.
Please nominate yourself **by March 25, 2026**.--->

> **Applications will open in May 2026.**

Reviewer: ```suggestion
**Applications will open (linked here) April 22, 2026.**
```

<!---Jump to [how to apply](#sec-how-to-apply), below.--->

[Openscapes](https://openscapes.org/) is an approach and a movement that helps
researchers find each other and feel empowered to conduct data-intensive science
[@lowndes2019; @robinson2022]. Since 2019, Openscapes has mentored over 600
researchers at universities across the US and at NASA and NOAA, helping to shift
institutional culture to develop climate solutions with open science
[@lowndes2024].

NASA Earthdata has volumes of remote sensing data collected by platforms in
space, air, water, and land. Substantial suborbital data is collected from
airborne sampling via airplanes, ground networks, and field sampling on land,
boats, and buoys.

> Suborbital science teams are teams that create and use suborbital data via NASA

Reviewer: make this part of the paragraph above

Author: Included in #241

programs like EVS-4. The science teams use suborbital data to ground-truth
satellite data, as well as to conduct other awesome research like studying
Arctic coastlines via the FORTE mission and glaciers and ice sheets via the
Snow4Flow mission. [Read about how Openscapes supports NASA suborbital science
teams](https://openscapes.org/blog/2026-03-03-suborbital/)

> Throuhg the champions program, EVS-4 and SoAR teams will develop common skills

Reviewer: ```suggestion
Through the Champions Program, EVS-4 and SoAR teams will develop common skills
```

> Through the Champions Program, EVS-4 and SoAR teams will develop common skills

Reviewer: delete this paragraph

Author: In ref #241

for documenting workflows with NASA suborbital data (Documentation, Open Science

> Data Management Plans, GitHub, “forking as a worldview” mindset). Working

Reviewer: ```suggestion
Data Management Plans, GitHub, “Forking as a Worldview” mindset). Working
```

together in the cohort will help to lower barriers for a better workflow
documentation by developing shared skills and strengthened relationships.

## Program details

[Openscapes Champions](https://www.openscapes.org/champions/) is a
remote-by-design mentorship program for researchers to explore open data science
practices. [Core lessons
include](https://openscapes.github.io/series/what-to-expect.html#cohort-calls)

> open mindset, Open Science Data Management Plans, GitHub for publishing and

Reviewer: ```suggestion
open mindset, GitHub for publishing and
```

> project management, documentation strategies, team culture, and metadata

Reviewer: ```suggestion
project management, coding and data strategies, and team culture.
```

> strategies.

Reviewer: ```suggestion
To support EVS-4 teams, this cohort will also focus on documentation strategies for suborbital data.
```

The 2026 NASA Suborbital Openscapes Champions Cohort will run in June 2026. We
will meet as a Cohort via Zoom four times over one month for 1.5 hours, on
alternating Wednesdays or Thursdays. We propose the following three options:

 - **When**:

>    - Option 1) Thursdays at 12-1:30 pm PT  Dates: June 4, 11, 18, 25

Reviewer: ```suggestion
```

>    - Option 2) Wednesdays at 10-11:30 am PT  Dates: June 3, 10, 17, 24

Reviewer: ```suggestion
   - Wednesdays at 10-11:30 am PT  Dates: June 3, 10, 17, 24
```

>    - Option 3) Thursdays at 10-11:30 am PT  Dates: June 4, 11, 18, 25

Reviewer: ```suggestion
```

> <!--- on alternating Wednesdays in April - June, 2026.

Reviewer: I'll leave this to you to uncomment and check formatting :)

> -   **Dates**: April 15, 29, May 13, 27, June 10.

Reviewer: ```suggestion
-   **Dates**: June 3, 10, 17, and 24.
```

Author: This change was already included. Unlike the other suggestions that doesn't appear as `resolved`

-   **Times**: 14:00 - 15:30 GMT.

-   **Where**: remotely, via Zoom. --->

-   **Who**: EVS-4 and SoAR teams.

-   **Cost**: Free. This opportunity is supported by NASA through a contract to Openscapes.

-   **Expected time commitment**: 2.5hrs/week for 1 month is an expected time 

>     commitment. This accounts for 1.5 hours/week of synchronous Zoom calls, and

Reviewer: ```suggestion
    commitment. This accounts for 1.5 hours/week of synchronous Zoom calls, plus self-organized time
```
